src/main/java/org/deepsymmetry/beatlink/VirtualRekordbox.java

```diff
@@ -1215,103 +963,6 @@ public DeviceUpdate getLatestStatusFor(int deviceNumber) {
         return null;
```

Comment:
Why is this returning `null`? Is the idea that people will call this method on the `VirtualCdj` instead like they are accustomed to? That makes sense, but if so this method can simply be removed.